File: Calories.App/Calories.App/Calories.App/Entities/Users/UserRoles.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using System.Text;

namespace Calories.App.Entities.Users
{
    public enum UserRoles
    {
        [EnumMember(Value = "admin")]
        Admin,
        [EnumMember(Value = "member")]
        Member,
        [EnumMember(Value = "manager")]
        Manager
    }
}

File: server/modules/users/entities/user-roles.js


/**
 * @enum
 * @description The standard user role, it can CRUD its own records.
 */
const UserRoles = {
    /** @description The administrator user role, it can CRUD all records ands users. */
    Admin: 'admin',
    /** @description The manager user role, it can CRUD member user records (the user only). */
    Manager: 'manager',
    /** @description The standard user role, it can CRUD its own records. */
    Member: 'member'
};

module.exports = UserRoles;